app/controllers/admin/items_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Admin::ItemsController < ApplicationController
+  def edit
+    @item = Item.find_by(slug: params[:id])
+  end
+
+  def update
+    item = Item.find_by(slug: params[:id])
+    if item.update(item_params)
+     flash[:success] = "Your updates have been saved"
+     redirect_to dashboard_path
+   else
+     flash.now[:warning] = @item.errors.full_messages.join(", ")
+     render :edit
+   end
+ end
+
+private
+
+  def item_params
+    params.require(:item).permit(
+    :name,
+    :price,
+    :description,
+    :category_id,
+    :image_url,
+    :status,
+    :slug
+    )
+  end
+end
```

Comment:
I'm not sure if we need an admin/items_controller anymore...I think we might just need to add before_action :business_admin? to the relevant routes in items_controller.